packages/forma-36-website/src/components/Header.js

```diff
@@ -12,32 +12,32 @@ const styles = {
     height: 70px;
     align-items: center;
   `,
-  logo: css`
-    display: flex;
-    flex: 1 0 0;
-    text-decoration: none;
-    color: #fff;
-  `,
   logoLink: css`
     display: flex;
     text-decoration: none;
     color: #fff;
   `,
-  // TODO: Sort out vertical text alignment
   logoText: css`
     font-weight: ${tokens.fontWeightDemiBold};
     font-size: ${tokens.fontSizeXl};
+    line-height: 32px;
```

Comment:
This scares me right now but I've not checked out why it's needed yet 😅

Reply:
Hmmm maybe I should change this to flex

Basically there was a TODO saying something like "figure out a way to center the logo"
In this line, I put a line-height of 'Forma 36' text equal to the svg height. But I will change to flex and then we don't have magic numbers. It will be future proof too